src/lib.rs

```diff
@@ -150,13 +153,13 @@ impl<'a> SelfEncryptor<'a> {
     }
   }
 
-  /// This is an implementation of the get_storage function from example
+  /// This is an implementation of the get_storage function from example.
   pub fn get_storage(&'a mut self) -> &'a mut Storage { self.storage }
 
-  /// Write method mirrors a posix type write mechanism
-  /// loosely mimics filsystem interface for easy connection to FUSE like
+  /// Write method mirrors a posix type write mechanism.
+  /// It losely mimics a filesystem interface for easy connection to FUSE like
```

Comment:
I already corrected this to 'loosely', and as a matter of style, it is okay to have elliptic descriptions in the benefit of faster readbility and less text - as a pure matter of taste maybe